R/create_cov_object.R

```diff
@@ -50,7 +56,8 @@ create_cov_object <- function(
     dat_i$t <- 0
   } else {
     if (!time_column %in% colnames(dat_i)) {
-      warning("missing time column: ", time_column)
+      cli::cli_warn(
+        paste0("Missing time column: ", time_column))
```

Comment:
style: closing parenthesis should be on a new line, or this should be all one line (i think it fits).